### PR TITLE
Nomis integration ui

### PR DESCRIPTION
**Pull request by ibrechin**

### CODE REVIEW

**Comment by ibrechin** [comment]
retest this please

File: mtp_cashbook/apps/cashbook/views.py [hunk 1]
```diff
@@ -247,3 +254,119 @@ def get_context_data(self, **kwargs):
...
+    title = _('New credits')
+    form_class = ProcessNewCreditsForm
+    template_name = 'cashbook/new_credits.html'
+    success_url = reverse_lazy('new-credits')
+
+    def get_form_kwargs(self):
+        form_kwargs = super().get_form_kwargs()
+        form_kwargs['request'] = self.request
+        if 'ordering' in self.request.GET:
+            form_kwargs['ordering'] = self.request.GET['ordering']
```
Comment (ushkarev) [inline mtp_cashbook/apps/cashbook/views.py]: should this value have some validation? does the api silently ignore invalid ordering?

File: mtp_cashbook/apps/cashbook/forms.py [hunk 2]
```diff
@@ -324,3 +330,255 @@ def get_query_data(self):
...
+
+    def _request_all_credits(self):
+        return retrieve_all_pages(
+            self.client.credits.get, status='available',
+            ordering=self.ordering
+        )
+
+    def clean_credits(self):
+        """
+        If discard button clicked => ignore any credits that might
```
Comment (ushkarev) [inline mtp_cashbook/apps/cashbook/forms.py]: i think this doc string is a lie

File: mtp_cashbook/templates/cashbook/change_notification.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+{% block inner_content %}
+
+  <header>
+    {% language_switch %}
+    <h1 class="heading-large">{% trans 'Digital cashbook is now connected to NOMIS' %}</h1>
+  </header>
+
+  <div class="mtp-change-notification__page">
+    <img class="mtp-change-notification__graphic" src="{% static 'images/nomis-change-graphic.png' %}"/>
+    <img class="mtp-change-notification__graphic2x" src="{% static 'images/nomis-change-graphic@2x.png' %}"/>
```
Comment (ushkarev) [inline mtp_cashbook/templates/cashbook/change_notification.html]: would be simpler to use a `srcset` attribute on one image element and avoid having to do any styles. like in the training section

File: mtp_cashbook/templates/cashbook/includes/nomis_print_dialog.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+{% load staticfiles %}
+
+<div id="print-dialog" class="Dialog" style="display: none">
+  <div class="Dialog-inner mtp-change-notification__dialog">
+    <h3 class="heading-medium">{% trans "Planning to enter these credits by hand?" %}</h3>
+    <img class="mtp-change-notification__graphic" src="{% static 'images/nomis-change-graphic.png' %}"/>
+    <img class="mtp-change-notification__graphic2x" src="{% static 'images/nomis-change-graphic@2x.png' %}"/>
```
Comment (ushkarev) [inline mtp_cashbook/templates/cashbook/includes/nomis_print_dialog.html]: would be simpler to use a `srcset` attribute on one image element and avoid having to do any styles. like in the training section

Comment (ibrechin) [inline mtp_cashbook/templates/cashbook/includes/nomis_print_dialog.html]: GOD I GOT IT THE FIRST TIME

File: mtp_cashbook/assets-src/stylesheets/views/_all_credits.scss [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    display: none;
+  }
+}
+
+.mtp-all-credits-search__filter-heading {
+  font-weight: bold;
+}
+
+.mtp-all-credits-search__inputs {
+  border: 2px solid #bfc1c3;
```
Comment (ushkarev) [inline mtp_cashbook/assets-src/stylesheets/views/_all_credits.scss]: `$border-colour`

**Comment by ibrechin** [comment]
change cookie timeout

File: mtp_cashbook/apps/cashbook/views.py (hunk 1 above)
Comment (ibrechin) [inline mtp_cashbook/apps/cashbook/views.py]: yes it does

**Pull request merged**